app/src/components/MemberDetail.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div className="card bg-white shadow-xl flex flex-row justify-between h-[83px] w-full">
```

Comment:
h-[83px] fixed height is not realy good. 

And you can use tailwind utility class to set the margin and the pading you want, instead of using height. 

And as advice it's not bad to have a big spacing. You can keep the default one. 
**With**
![image](https://github.com/globe-and-citizen/cnc-portal/assets/28951070/c14af5d0-bea4-44bb-9fb6-6de8cd7db149)

**Whitout** 
![image](https://github.com/globe-and-citizen/cnc-portal/assets/28951070/83c3cd0e-42d5-4f1d-baad-082c7f00901d)